Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py

```diff
@@ -156,7 +156,7 @@ def main():
         else:
             readable_output = NO_CAMPAIGN_INCIDENTS_MSG
 
-        return_results(CommandResults(readable_output=readable_output))
+        return_results(CommandResults(readable_output=readable_output, raw_response=incidents))
```

Comment:
relevant for all cases:
those are widget scripts - are you sure the intention is to return the incident list and not the markdown table (`readable_output`)?
did you test it?